huaweicloud/services/lts/resource_huaweicloud_lts_waf_access.go

```diff
@@ -247,11 +249,22 @@ func buildDeleteWAFAccessBodyParams() map[string]interface{} {
 	return bodyParams
 }
 
+func isUUIDFormat(uuid string) bool {
+	pattern := "[0-9a-fA-F]{8}(-[0-9a-fA-F]{4}){3}-[0-9a-fA-F]{12}"
+	match, _ := regexp.MatchString(pattern, uuid)
+	return match
+}
```

Comment:
```suggestion
func isUUIDWIthHyphens(uuid string) bool {
	pattern := "[0-9a-fA-F]{8}(-[0-9a-fA-F]{4}){3}-[0-9a-fA-F]{12}"
	match, _ := regexp.MatchString(pattern, uuid)
	return match
}
```
pls bring this function to the utils package,